docs/DeveloperGuide.adoc

```diff
@@ -315,29 +313,68 @@ At first, the `execute()` function of the `LogicManager` class is called, which
 **Cons:** The keywords have to be exactly the save i.e friend is not the same as friends. +
 **Alternative 2:** Use string.contains() to find tasks. +
 **Pros:** Keywords do not have to be exactly the same as descriptions in tasks. +
 **Cons:** The search result might not be accurate if the user search using the keyword 'a' all the tasks'descriptions that contains the character 'a' will appear.
 
-== ListTask mechanism
+=== ListTask mechanism
 
 The ListTask mechanism is facilitated by the `ListTaskCommand` class which inherits from `Command` class. It allows the user to list out all the current tasks.
 
 Suppose the user has just executed the `ListTaskCommand` using `listtask`.
 
 At first, the `execute()` function of the `LogicManager` class is called, which goes on to pass the command string into the `parseCommand()` function of the `AddressBookParser` class. This, in turn, determines that the `ListTaskCommand` feature is being evoked and calls the `execute()` in the `ListTaskCommand` class which update the current list to show all tasks creates and return a `CommandResult` object.
 
-== SelectTask mechanism
+=== SelectTask mechanism
 
 The SelectTask mechanism is facilitated by the `SelectTaskCommand` class which inherits from `Command` class. It allows the user to select the task at specific index.
 
 Suppose the user has just executed the `SelectTaskCommand` using `selecttask`.
 
 At first, the `execute()` function of the `LogicManager` class is called, which goes on to pass the command string into the `parseCommand()` function of the `AddressBookParser` class. This, in turn, determines that the `SelectTaskCommand` feature is being evoked and calls the `SelectTaskCommandParser` to parse the string and check if the index is valid. After validation the `SelectTaskCommand` class is been called. The `LogicManager` then calls the `execute()` in the `SelectTaskCommand` class which calls `EventCenter` with the parameter of `JumpToTaskListRequestEvent` class to highlight selected task card in Ui at target index. Then `execute()` creates and return a `CommandResult` object.
 
+// tag::calendar[]
+=== Calendar mechanism
+
+The calendar mechanism is facilitated by the `DatePickerSkin` skin instance which capture the pop-up content from the `DatePicker` control in `JavaFX`. It allows the user to have a calendar view of all contacts' `Birthday` and tasks' `Deadline`.
+
+Suppose the address book is modified (e.g. `add`, `edit`, `addtask`, `deletetask`).
+
+After the modification of the address book i.e. saving of data to the `addressbook.xml` file, `ObservableList` of `Person` and `Task` being stored in the address book are passed into the `CalendarPanel` Ui.
+
+Next, `getDayCellFactory` is being evoked, which will format the `DateCell` of the `DatePicker`. The formatting is done with `Birthday` and `Deadline` as conditions and, colour and tooltip of `DateCell` as variable to change.
+
```

Comment:
Remove the "being"
No comma after and.
*variable
What does "variable to change" mean? They can be changed manually by a user or they are variables that will be changed by the developer later? Elaborate a little more maybe.